playbooks/ContinuousDelivery/image_promotion.adoc

```diff
@@ -0,0 +1,117 @@
+= OpenShift Enterprise 3 Image Promotion Strategies
+Andrew Block <ablock@redhat.com>
+v1.0, 2015-10-05
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
+
+This document provides strategies for promoting an application between various OpenShift environments. The level of segregation of how environments are divided are at the discretion of the user. It is out of the scope of this particular document on how the application is built and packed into the initial image
+
+== High Level Components
+
+Each strategy defined within this document will have several consistent themes:
+ 
+* 1:x OpenShift Environments
+* Integration environment
+** It is _recommended_ that the Jenkins CI server be used to facilitate the promotion process, however standalone scripts may be used instead if desired
+Docker client to manipulate images
+The user executing the scripts must be a member of the docker group in order to access the Docker socket. Additional information on this topic can be found https://docs.docker.com/installation/fedora/[here]
+
+== Prerequisites
+
+The majority of the work involves the integration server and then ability to access and communicate with both OpenShift Environments. To accomplish this, the integrated docker registry must be externally exposed for invocations by external parties. The process of externally exposing the integrated docker registry is described in External Access of link:../Installation/expose_docker_registry.adoc[Integrated Docker Registry].Ensure the necessary software and docker client configurations are completed before proceeding.
```

Comment:
I think we may want a List here outlining all of the components involved in the solution we're describing.. i.e.:
-build server
-openshift environment(s)
-exposed registry
-router
-...etc...